fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/projects/sandpiles-opportunity.md

attribution: Basekamp & friends
images:
  - src: 32.jpg
    alt: Sand sculpture installation in wooden box with multiple mounds and valleys marked with small flags and labels, people visible in background at Versionfest
  - src: 25.jpg
    alt: Close-up aerial view of sand landscape with tall peaked structures and small colored flags planted throughout the terrain
  - src: 26.jpg
    alt: Sand installation with varied topography including pyramidal forms and flat areas marked with small white flags, person visible in blurred background
  - src: 64.jpg

>     alt: Detailed view of carved sand block showing ribbed horizontal texture pattern with small flag marker protruding from top

Reviewer: ```suggestion
    alt: Detailed view of sand house structure showing raked texture pattern on roof with small flag marker protruding from top
```

  - src: 65.jpg

>     alt: Pair of green Versionfest branded sunglasses partially buried in sandy surface alongside other small markers

Reviewer: ```suggestion
    alt: Pair of green cloth badges partially buried in sand mound within sandbox
```

  - src: 66.jpg
    alt: Business card reading "A+C=T" artlab with colorful letter logo, surrounded by plastic fork, spoon, straws and white egg in sand
  - src: 67.jpg
    alt: Fractal Workspace business card standing upright in sand structure showing contact information for Chicago location
  - src: 72.jpg

>     alt: Another view of Fractal Workspace business card embedded in sand mound with red marker visible in background

Reviewer: ```suggestion
    alt: Topdown view of Fractal Workspace business card embedded in sand building structure with other markers in surrounding sand
```

  - src: 74.jpg

>     alt: Section of sand installation featuring bright yellow ball, green container, and various small markers and objects

Reviewer: ```suggestion
    alt: Section of sand installation featuring bright yellow ball, green badge, and various small markers and objects
```
